internal/pkg/utils/etcdhelper/fortest.go

```diff
@@ -108,13 +108,11 @@ func ClientForTestFrom(t testOrBenchmark, endpoint, username, password, namespac
 
 	// Cleanup namespace after the test
 	t.Cleanup(func() {
-		_, err := originalKV.Delete(ctx, namespaceStr, etcd.WithPrefix())
+		_, err := originalClient.Delete(ctx, namespaceStr, etcd.WithPrefix())
 		if err != nil {
 			t.Fatalf(`cannot clear etcd namespace "%s" after test: %s`, namespaceStr, err)
 		}
-		if err := etcdClient.Close(); err != nil {
-			t.Fatalf(`cannot close etcd connection after test: %s`, err)
-		}
+		cancel()
```

Comment:
Ukoncime kontext namiesto volania `Close()`.

Generovalo to `warnings` v testoch.

Myslim, ze je to drobny bug v etcd klientovi.

![image](https://user-images.githubusercontent.com/19371734/207635878-69b55b23-7afa-48d6-9814-9f7122986a4b.png)


Tiez tam volaju `cancel()`, ale hned aj `Close` nad `Watcher` a `Lease`.

No a `Watcher` a `Lease` posielaju este nejake data typu `odpajam sa ....`,
ale kedze je ctx skonceny, tak to niekedy nestihne dobehnut a zaloguje sa warning.

Teda my iba zavolame context cancel a nechame klienta nech sa ukonci sam.